src/DssSpell.t.base.sol

```diff
@@ -292,21 +292,26 @@ contract DssSpellTestBase is Config, DssTest {
             return;
         }
 
-        uint256 low = 5273074; // MCD_VAT Deployed Aug-06-2021 04:26:38 PM +UTC
+        string memory spellBlockPath = "./spellblock.txt";
+
+        uint256 low = vm.parseUint(vm.readLine(spellBlockPath));
```

Comment:
wondering if the `spellblock.txt` file should be closed after reading via `vm.closeFile` cheatcode to reset the offset and allow to read it from the beginning with `vm.readLine` following [docs](https://book.getfoundry.sh/cheatcodes/fs?highlight=cheat#file-cheat-codes).

Reply:
I see your point, but since its just a one line file and we only read once here, I'm not sure we need to add this.

Reply:
yeah, I mentioned for correctness, not sure what would happen when it will get read again.